fbw-a32nx/src/systems/shared/src/MagVar.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2021-2023 FlyByWire Simulations
+//
+// SPDX-License-Identifier: GPL-3.0
+
+import { Coordinates } from 'msfs-geo';
+
+export class MagVar {
+    static magneticToTrue(magneticHeading: Degrees, amgVar?: Degrees): Degrees {
+        return (720 + magneticHeading + (amgVar || SimVar.GetSimVarValue('MAGVAR', 'degree'))) % 360;
+    }
+
+    static trueToMagnetic(trueHeading: DegreesTrue, magVar?: Degrees): Degrees {
+        return (720 + trueHeading - (magVar || SimVar.GetSimVarValue('MAGNAR', 'degree'))) % 360;
+    }
+}
```

Comment:
This whole file should be deleted. It has some bugs (MAGNAR) and it should not be used anyhow. Use A32NX_Util.trueToMagnetic and A32NX_Util.magneticToTrue instead.